types/result.go

```diff
@@ -102,6 +103,7 @@ func (r *ResponseCheckTx) Result() Result {
 		Code: r.Code,
 		Data: r.Data,
 		Log:  r.Log,
+		Tags: r.Tags,
```

Comment:
There is no meaning to tags in checktx, as they should never be indexed. We should remove that from the api, as it would be misleading.